integration/raft/channel_participation_test.go

```diff
@@ -140,13 +140,14 @@ var _ = Describe("ChannelParticipation", func() {
 
 			By("ensuring the channel is usable by submitting a transaction to each member")
 			env := CreateBroadcastEnvelope(network, peer, "participation-trophy", []byte("hello"))
-			for _, o := range members {
+			for i, o := range members {
 				By("submitting transaction to " + o.Name)
+				i++
 				expectedChannelInfoPT.Height++
 				resp, err := ordererclient.Broadcast(network, o, env)
 				Expect(err).NotTo(HaveOccurred())
 				Expect(resp.Status).To(Equal(common.Status_SUCCESS))
-				expectedBlockNumPerChannel := map[string]int{"participation-trophy": int(expectedChannelInfoPT.Height - 1)}
+				expectedBlockNumPerChannel := map[string]int{"participation-trophy": i}
```

Comment:
I know you didn't introduce this code (guilty as charged) but can we do the similar fix here too?

Reply:
:broom